web/packages/dmt-core/src/components/AccessControl/AccessControlList.tsx

```diff
@@ -64,6 +64,7 @@ export const AccessControlList = (props: {
   const [loading, setLoading] = useState<boolean>(false)
   const [loadingACLDocument, setLoadingACLDocument] = useState<boolean>(false)
   const [tokenWithReadAccess, setTokenWithReadAccess] = useState<string>('')
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore

Reply:
How is this fixed in the master branch? ) This code is rebased against master and without // @ts-ignore gets errors.

Reply:
i think it depends on the version of react-oauth2-code-pkce is used. if using the latest, the problem should be fixed.
Maybe it will work to run yarn install locally?

Reply:
okey, we can remove the comments in another PR